launch/doc/.gitignore

```diff
@@ -1 +0,0 @@
-build
```

Comment:
https://github.com/ros2/rmw_fastrtps/pull/855#discussion_r2545027166

probably we want to wait until the above discussion result?

Reply:
i removed those files which look like a standalone sphinx-project (without using rosdoc2)